src/components/quote/StockQuoteWidget.tsx

```
@@ -5,56 +5,209 @@
+              }));
+          setNews(headlines);
+        } else {
+          setError('news', 'Failed to load news');
+        }
+      } catch {
+        if (cancelled) return;
+        setError('snapshot', 'Network error');
+        setError('profile', 'Network error');
+        setError('news', 'Network error');
+      } finally {
+        if (cancelled) return;
```

Biome (2.4.7)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)